cdap-ui/app/features/dashboard/templates/staticdashboard.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" ng-controller="WidgetColCtrl">
-  <div ng-class="col-xs-12 col-md-6"
+  <div ng-class="{'col-xs-12 col-md-6' : true}"
```

Comment:
if this is always true, then we should change this to just class="col-xs-12 col-md-6"  instead of ng-class